pkg/api/server/rest/authentication/bearertoken/bearertoken.go

```diff
@@ -0,0 +1,55 @@
+package bearertoken
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/gin-gonic/gin"
+	"github.com/mlab-lattice/lattice/pkg/api/server/rest/authentication"
+	"github.com/mlab-lattice/lattice/pkg/api/server/rest/authentication/user"
+)
+
+type Authenticator struct {
+	token authentication.Token
+}
+
+func New(token authentication.Token) (*Authenticator, error) {
```

Comment:
personally i might call `token` `authenticator`. `token` on my read kind of sounded like you were passing in a token not a token authenticator